homeassistant/components/unifiprotect/entity.py

```diff
@@ -106,12 +128,24 @@ def async_all_device_entities(
 
     if ufp_device is None:
         return (
-            _async_device_entities(data, klass, ModelType.CAMERA, camera_descs)
-            + _async_device_entities(data, klass, ModelType.LIGHT, light_descs)
-            + _async_device_entities(data, klass, ModelType.SENSOR, sense_descs)
-            + _async_device_entities(data, klass, ModelType.VIEWPORT, viewer_descs)
-            + _async_device_entities(data, klass, ModelType.DOORLOCK, lock_descs)
-            + _async_device_entities(data, klass, ModelType.CHIME, chime_descs)
+            _async_device_entities(
+                data, klass, ModelType.CAMERA, camera_descs, unadopted_descs
+            )
+            + _async_device_entities(
+                data, klass, ModelType.LIGHT, light_descs, unadopted_descs
+            )
+            + _async_device_entities(
+                data, klass, ModelType.SENSOR, sense_descs, unadopted_descs
+            )
+            + _async_device_entities(
+                data, klass, ModelType.VIEWPORT, viewer_descs, unadopted_descs
+            )
+            + _async_device_entities(
+                data, klass, ModelType.DOORLOCK, lock_descs, unadopted_descs
+            )
+            + _async_device_entities(
+                data, klass, ModelType.CHIME, chime_descs, unadopted_descs
+            )
         )
 
     descs = []
```

Comment:
This could probably use a refactor at some point to get rid of the linear searching

Reply:
It is only for a single device (no loop) so it is probably not a big deal at the moment to fix.